app/views/users/show.html.erb

```diff
@@ -1 +1,49 @@
-<h1>User's page</h1>
\ No newline at end of file
+<%= stylesheet_link_tag ('home') %>
+<section class="users-home">
+  <div class="user-main">
+    <div class="photo">
+      photo
+    </div>
+    <div class="name">
+      <ul>
+        <li>
+          Username: <%= @user.name %>
+        </li>
+      </ul>
+        <p class="user-activity">
+          <%= link_to "view post", user_path(id: @user.id) %>
+        </p>
+    </div>
+  </div>
+  <div class="bio">
+    <div class="bio-body">
+      <h5>Bio</h5>
+      <p>
+        <%= @user.bio %>
+      </p>
+    </div>
+  </div>
+  
+  <div class="post">
+    <% @user.posts.each_with_index do |post, index| %>
+    <div class="post-body">
+      Post#<%= index %>
+      <h3>
+        Post title:
+        <%= post.title %>
+      </h3>
+      <p>
+        <%= post.text %>
+      </p>
+      <div class="post-activity">
+        Comments: <%= post.comment_counter %>,
+        Likes:<%= post.likes_counter %>
+      </div>
+    </div>
+    <% end %>
+  </div>
+    <div class="all-posts">
+      <button type="button" class="all-post-btn">See all posts</button>
```

Comment:
- [ ] Kindly make sure that the `See all posts` button opens the user post index, where all the posts should be listed